config/initializers/training_routes.rb

```diff
@@ -82,3 +88,33 @@
     allocation_subjects: %w[Biology],
   ),
 ].freeze
+
+TRAINING_ROUTES_INITIATIVES = {
```

Comment:
I imagine that when we come to saving the initiative onto the trainee, it will be an enum with the four values:
- "Transition to Teach"
- "Now teach"
- "Maths & Physics Chairs Programme/Researchers in Schools"
- "Future Teaching Scholars"

It probably makes sense to store these four strings in a constant and then use in this `TRAINING_ROUTES_INITIATIVES` mapping too.

Reply:
Added enums